### PR TITLE
Projectkk2glider/new jitter filter for next2

### PR DESCRIPTION
**Pull request by projectkk2glider**
This pull addresses two things:
 * better jitter filter (the equivalent for master is in #3250)
 * improved Horus ADC driver

Tested on my Hours.

### CODE REVIEW

File: radio/src/targets/horus/adc_driver.cpp [hunk 1]
```diff
@@ -133,11 +133,11 @@ void adcInit()
 #define SPI_S1                         4
 #define SPI_6POS                       5
 #define SPI_S2                         6
-#define SPI_S3                         7
-#define SPI_S4                         8
+#define SPI_LS                         7
```
Comment (bsongis) [inline radio/src/targets/horus/adc_driver.cpp]: Why? On mine they are labelled S3 / S4. Is it different on yours?

Comment (projectkk2glider) [inline radio/src/targets/horus/adc_driver.cpp]: I have `L1` on the left linear slider (left of the power button) and `L2` label on the right one. I see that old photos had S3 and S4 there. I did not know that.

Comment (projectkk2glider) [inline radio/src/targets/horus/adc_driver.cpp]: I see that the trim labels have also changed. I will post a picture of mine.

**Comment by projectkk2glider** [comment]
dsc_9367

Here is my X12S, T1 and T4 changed places.

**Comment by bsongis** [comment]
Right it's different from mine. Also T1/T4 are swapped. T2/T3 are swapped.

**Comment by projectkk2glider** [comment]
Perhaps ADC channels also changed, because I corrected the ADC positions of L1/l2 and LS/RS also.

**Pull request merged**